content/en/docs/concepts/specs-requirements.md

```diff
@@ -0,0 +1,8 @@
+---
+title: "Notary v2 project specifications and requirements"
```

Comment:
`Notary v2` ==> `Notary`, since we don't use/reference `Notary v2` anymore.

Reply:
fixed